oarepo_vocabularies/ui/theme/assets/semantic-ui/js/oarepo_vocabularies_ui/search/components/NewItemButton.jsx

```diff
@@ -0,0 +1,21 @@
+import React, { useContext } from "react";
+import { Button } from "semantic-ui-react";
+import { SearchConfigurationContext } from "@js/invenio_search_ui/components";
+import { i18next } from "@translations/oarepo_vocabularies_ui/i18next";
+
+export const NewItemButton = () => {
+  const config = useContext(SearchConfigurationContext);
```

Comment:
We shouldn't use search config just to get create link, we should use records create ui link

Reply:
Ciao Miro, but this is another app and I don't have access to useFormContext here. Could you please show me what you mean?

Reply:
We will use link provided by https://github.com/oarepo/oarepo-ui/pull/58 here

Reply:
And will update this code in a different PR when that issue gets resolved